src/router.js

```diff
@@ -8,6 +8,23 @@ const messages = {
     message: 'Email ou mot de passe incorrect.',
   },
 };
+const DEFAULT_EIDAS_LEVEL = 'eidas3';
+
+export const getEidasLevel = (req) => {
+  if (!req.session || !req.session.infos || !req.session.infos.acr_values) {
```

Comment:
could plz tell me where this session come from ? :tea: I think reading the query params should be enough